frontend/components/side_panels/SiteNavSidePanel/navItems.js

```diff
@@ -1,27 +1,30 @@
+import PATHS from 'router/paths';
+import URL_PREFIX from 'router/url_prefix';
+
 export default (admin) => {
   const adminNavItems = [
     {
       icon: 'admin',
       name: 'Admin',
       location: {
-        regex: /^\/admin/,
-        pathname: '/admin/users',
+        regex: new RegExp(`^${URL_PREFIX}/admin/`),
```

Comment:
This, and more, break if URL_PREFIX has a trailing slash.

I think thats enforced by the regex, but it feels weirdly fragile to me

Reply:
Do you have some idea of how else we could handle this besides at the entry point of the program? I would not like to enforce or strip this at each point it is used.

Reply:
See above.